### DISCUSSION TITLE
Illogical behavior of handcuffs

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13232] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

1. Flashlights, glasses, headphones do not stack, but handcuffs do.
2. If you transfer a stack of handcuffs to another character’s inventory and activate them, the entire stack is blocked. When you try to separate this stack, the handcuffs are automatically removed.
Solution 1:
Make sure that handcuffs can be stored only without a stack, like glasses, headphones, flashlights.
Solution 2:
Stack handcuffs like balloons, only in the closet.

### Reproduction steps

1. Take stack of handcuff and place it into bot/player inventory
2. Activate it
3. All stack is blocked as if you use several handcuffs on his hands.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.2.7.0 (Winter Update hotfix)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Solution 3: don't unlock handcuffs when moving them in inventory

**Comment by pavel4win** [comment]
> Solution 3: don't unlock handcuffs when moving them in inventory

This is not whole solution but good design idea.
Main problem is stack, not unlock. The glasses even smaller then handcuffs. Why glasses not stacked then?

**Comment by Regalis11** [comment]
Thank you for the report!

> Main problem is stack, not unlock. The glasses even smaller then handcuffs. Why glasses not stacked then?

In this case realism is being stretched a bit for convenience's sake: security officers often need to carry several pairs of handcuffs. I agree this, and the behavior when moving items away from a stack like this, is somewhat illogical, but I would argue it's not a bug, so I'll move this to discussions.
